services/networking/vpc/threats.yaml

```diff
@@ -1,78 +1,64 @@
 common_threats:
-  - CCC.TH01 # Access control is misconfigured
-  - CCC.TH02 # Data is intercepted in transit
-  - CCC.TH03 # Deployment region network is untrusted
-  - CCC.TH06 # Data is lost or corrupted
-  - CCC.TH07 # Logs are Tampered With or Deleted
+  - CCC.TH01  # Access control is misconfigured
+  - CCC.TH02  # Data is intercepted in transit
+  - CCC.TH03  # Deployment region network is untrusted
+  - CCC.TH06  # Data is lost or corrupted
+  - CCC.TH07  # Logs are Tampered With or Deleted
 
 threats:
   - id: CCC.VPC.TH01
     title: Unauthorized Access via Insecure Default Networks
     description: |
-      Default network configurations may include insecure settings and open firewall rules,
-      leading to unauthorized access and potential data breaches.
+      Default network configurations may include insecure settings and open
+      firewall rules,leading to unauthorized access and potential data
+      breaches.
     features:
-      - CCC.VPC.F01 # Custom Network Creation
-      - CCC.F06 # Identity Based Access Control (common feature)
+      - CCC.VPC.F01  # Custom Network Creation
     mitre_technique:
-      - T1040 # Network Sniffing
-      - T1136 # Create Account
+      - T1040  # Network Sniffing
 
   - id: CCC.VPC.TH02
-    title: Exposure of Virtual Machines to Public Internet
+    title: Exposure of Resources to Public Internet
     description: |
-      Assignment of external IP addresses to virtual machines exposes them to the public internet,
-      increasing the risk of attacks such as brute force, exploitation of vulnerabilities, or unauthorized access.
+      Assignment of external IP addresses to resources exposes resources to the
+      public internet, increasing the risk of attacks such as brute force,
+      exploitation of vulnerabilities, or unauthorized access.
     features:
-      - CCC.VPC.F02 # External IP Address Assignment
-      - CCC.F01 # Encryption in Transit Enabled by Default (common feature)
+      - CCC.VPC.F04 # Public Subnet Creation
     mitre_technique:
-      - T1133 # External Remote Services
-      - T1078 # Valid Accounts
+      - T1133  # External Remote Services
+      - T1078  # Valid Accounts
```

Comment:
While valid accounts could lead to exposure, T1046 (Network Service Discovery) is a more specific mapping. It's commonly performed to identify exposed resources.